src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,71 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(p -> Arrays.stream(p.split(",")))
+                .map(Integer::valueOf)
+                .mapToInt(i -> i)
+                .filter(i -> i % 2 == 0)
+                .min().orElseThrow(() -> new RuntimeException(
```

Comment:
Every operation has to be on new line
```suggestion
                .min()
                .orElseThrow(() -> new RuntimeException(
```